routes/web.php

```diff
@@ -22,13 +27,20 @@
 
 Auth::routes();
 
-Route::get('/home', [App\Http\Controllers\HomeController::class, 'index'])->name('home');
+// Route::get('/home', [App\Http\Controllers\HomeController::class, 'index'])->name('home');
 
 
 
 // Route Group
 Route::group(['middleware' => 'auth'], function () {
     # LOGINED USERS ONLY
+    Route::group(['prefix'=>'users','as'=>'users.'], function(){
+        Route::get('/', [HomeController::class, 'index'])->name('top');
+        Route::get('/reserved/show', [HomeController::class, 'show'])->name('reserved.show.details');
+        Route::get('/profile/{id}/show', [ProfileController::class, 'show'])->name('profile.show');
+        Route::get('/profile/edit', [ProfileController::class, 'edit'])->name('profile.edit');
+        Route::patch('/profile/update', [ProfileController::class, 'update'])->name('profile.update');
```

Comment:
for `/profile/{id}/show`, change that to `/{user}`
for `/profile/edit`, change that to `/{user}/edit`
for `/profile/update`, change that to  just `/`